### PR TITLE
feat: extension as side panel

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/router/index.ts [hunk 1]
```diff
@@ -155,7 +155,10 @@ if (IS_MOBILE_APP) {
       }
     });
 
+    // TODO establish if we still need this
     router.afterEach((to) => {
+      console.log('PTH', to.path);
```
Comment (martinkaintas) [inline src/popup/router/index.ts:160]:
```suggestion
```

**Review by martinkaintas — CHANGES_REQUESTED** [review]
Also would be nice if the extension is open as side-panel that we don't open new popups but show the modal on the side-panel

File: src/popup/pages/Names/AuctionList.vue [hunk 2]
```diff
@@ -154,7 +154,6 @@ export default defineComponent({
 
 .auction-list {
   .toolbar {
-    --background: var(--screen-bg-color);
```
Comment (martinkaintas) [inline src/popup/pages/Names/AuctionList.vue:157]: image

Comment (peronczyk) [reply src/popup/pages/Names/AuctionList.vue:157]: OK, I brought back the variable.

**Review by martinkaintas — APPROVED** [review]
lgtm

**Pull request merged**